tools/serverpod_cli/lib/src/analyzer/entities/yaml_definitions/class_yaml_definition.dart

```diff
@@ -54,7 +54,7 @@ class ClassYamlDefinition {
                   Keyword.relation,
                 },
                 alternativeUsageMessage:
-                    'Use the relation keyword instead. E.g. relation(parent=parent_table)',
+                    'Use the relation keyword instead. E.g. relation(parent=parent_table). Note that the onDelete default has changed from "Cascade" to "NoAction" to preserve the existing behavior you also need to add relation(onDelete=Cascade).',
```

Comment:
Suggestion: Improve hint message.

```suggestion
                    'Use the relation keyword instead. E.g. relation(parent=parent_table). Note that the default onDelete action changes from "Cascade" to "NoAction" when using the relation keyword.',
```

Reply:
Suggestion to use a similar message for the parent field.